src/routes/e2e/widget/inline.svelte

```diff
@@ -17,21 +17,30 @@
   journey.onSuccess((response: User) => (userInfo = response));
   journey.onFailure((error: string) => console.log(error));
 
-  s_onMount(() => {
+  onMount(async () => {
+    let content;
+    /**
+     * Reuse translated content from locale api if not en-US
+     */
+    if (navigator.language !== 'en-US') {
+      const response = await fetch(`${window.location.origin}/api/locale`);
+      content = response.ok && (await response.json());
+    }
     // TODO: Add method to refresh form
     new Widget({
       target: formEl,
       props: {
         config: {
           clientId: 'WebOAuthClient',
-          redirectUri: 'https://localhost:3000/callback',
+          redirectUri: `${window.location.origin}/callback`,
           scope: 'openid profile email me.read',
           serverConfig: {
             baseUrl: 'https://openam-crbrl-01.forgeblocks.com/am/'
           },
           realmPath: 'alpha',
           tree: 'Login',
         },
+        content,
```

Comment:
This is the content object passed into the Widget initialization. This is where we are using the locale API endpoint.